cli/tests/pcluster/config/test_config_patch.py

```diff
@@ -505,6 +505,43 @@ def _test_more_target_sections(base_conf, target_conf):
     target_storage["EbsSettings"]["Iops"] = 20
     target_storage["EbsSettings"]["VolumeType"] = "gp2"
 
+    # Check the update of Ebs,Efs,FsxLustreSettings
+    # with both updatable and not updatable keys
+    for storage_type in ("Ebs", "Efs", "FsxLustre"):
```

Comment:
Thanks for adding these unit tests!

The `_test_more_target_sections` function was used to verify that a generic case on which you were adding a section is working as expected.

I think we can move these new tests outside of `_test_more_target_sections` to keep the test more self contained and you can have a new generic `_test_storages`.

In this way we can continue to extend storage specific unit tests in this new function. What do you think?

Reply:
Done